README.md

```diff
@@ -177,10 +177,14 @@ A shopcart list can be got with a `GET` request on `'/shopcarts'`. The response
     }
 ]
 ```
+## Updating a Shopcart Item
+A shopcart item can be updated with a `PUT` request on `'/shopcartitems'`.
```

Comment:
Should be /shopcartitems/id 
And include that of the field(s) to be included in the content for the PUT request, such as:
```json
{
    "amount": 4,
}
```